### ISSUE TITLE
Category view results in Access Denied error

### ISSUE THREAD

**Issue by deki** [aws-samples/aws-bookstore-demo-app#14]
**Describe the bug**
The category view e.g. https://mydistname.cloudfront.net/category/Cars results in an Access Denied error.

`<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>012345</RequestId><HostId>R2oclEoKzgck7Jw/5rBKjXDjqKV0bOtBZrBzfttfnvanUu9KK/kbZmeOWXdrz6X9IGVfSk=</HostId></Error>`

**To Reproduce**
1. Navigate to Index page
2. Click on one of the categories in the Navigation bar
3. AccessDenied error will appear

**Expected behavior**
Books in the selected category are shown

**Comment by smohiuddin** [comment]
Hi, we are unable to reproduce this error. Sometimes CloudFront distributions can take as much as 30 min to start up. Are you still seeing this issue?

**Comment by deki** [comment]
Yeah it happens even after days but only when using the main navigation bar. For example clicking Cookbooks in the navigation bar fails with AccessDenied while clicking the "Browse cookbooks" link successfully displays the page.

**Comment by triggan** [comment]
After testing this in the stack, I was successfully able to reproduce this issue and apply the custom error response as noted in the following article.  I need to apply this to the CFN stack so that this error response is created during a deployment:  https://www.codebyamir.com/blog/fixing-403-access-denied-errors-when-hosting-react-router-app-in-aws-s3-and-cloudfront
